index.html

```diff
@@ -110,6 +182,7 @@ <h2 class="tonic4">Multi-Post Stories</h2>
 					<li class="btn34">javaScript</li>
 				</ul>
 				<button class="btn44" type="submit">See Project</button>
+				</div>
 			</div>
 		</section>
```

Comment:
- [ ] Please decrease the form fields width size and make sure it matches the Design.

### Your design
![image](https://user-images.githubusercontent.com/83803414/186702156-ebab4d1b-d45d-491e-b759-39dc229bda79.png)

### Figma design
![image](https://user-images.githubusercontent.com/83803414/186702223-509b0b49-ad15-406d-9498-3187aeed7f3f.png)